service/templates/confirm_selection.html

```diff
@@ -14,6 +14,14 @@ <h1 class="heading-large collapse-top">Confirm your order</h1>
         {% else %}
         {{ _('Not Available') }}
         {% endif %}
+        {% if params['title']['property_notes'] %}
```

Comment:
if property_notes is null then it won't fall into this if statement. Jinja's if mirrors Python's if behaviour meaning that null / None is falsy. See https://docs.python.org/2.4/lib/truth.html